### PR TITLE
Updated the readme .

### PR DESCRIPTION
**Pull request by Nafisarkar**
Added a banner also update the readme with instruction . button for download and joining whatsapp

### CODE REVIEW

**Review by aniruddha-adhikary — CHANGES_REQUESTED** [review]
Thank you so much for choosing to contribute!

Overall looks great. I suppose the text is a bit Chat-GPT-esque. A little rewrite may be in order.

File: README.md [hunk 1]
```diff
@@ -1,15 +1,27 @@
...
+### Get Started
+[![Download the latest release](https://shields.io/badge/Download-Latest%20Release-blue?style=for-the-badge)](https://github.com/aniruddha-adhikary/mrt-buddy/releases)
 
-100% UNOFFICIAL COMMUNITY EFFORT.
+Android App for Dhaka MRT Card Balance Check
+------------------------------------------
+### Overview
 
-## How does it work?
+MRT Buddy is an unofficial community-driven Android app designed to check the balance of your Dhaka MRT Card.
```
[the remark this answers is not in the document]
Comment (Nafisarkar) [reply README.md:12]: Sure

File: README.md [hunk 2]
```diff
@@ -1,15 +1,27 @@
...
+[![MRT Buddy Banner](https://i.ibb.co.com/9wf4KwD/Mrtbuddy-Banner.jpg)](https://github.com/aniruddha-adhikary/mrt-buddy/releases)
 
-Android app to check the balance of your Dhaka MRT Card.
 
-Download here => https://github.com/aniruddha-adhikary/mrt-buddy/releases
+### Get Started
+[![Download the latest release](https://shields.io/badge/Download-Latest%20Release-blue?style=for-the-badge)](https://github.com/aniruddha-adhikary/mrt-buddy/releases)
 
-100% UNOFFICIAL COMMUNITY EFFORT.
+Android App for Dhaka MRT Card Balance Check
```
[the remark this answers is not in the document]
Comment (Nafisarkar) [reply README.md:8]: i suppose 😐

File: README.md [hunk 3]
```diff
@@ -1,15 +1,27 @@
...
-## How does it work?
+MRT Buddy is an unofficial community-driven Android app designed to check the balance of your Dhaka MRT Card.
 
-The FeliCa NFC Card stores the last 10 transactions performed by your card. That part of the 
-memory is readable through NFC commands. Thus, the system does not require any API connectivity
-to servers. However, only limited to 10 transactions. 
+### Key Features
+
+* Reads the last 10 transactions from the FeliCa NFC Card
+* No API connectivity required, ensuring a seamless user experience
```
[the remark this answers is not in the document]
Comment (Nafisarkar) [reply README.md:17]: Removed !

File: README.md [hunk 4]
```diff
@@ -1,15 +1,27 @@
...
 
-## How does it work?
+MRT Buddy is an unofficial community-driven Android app designed to check the balance of your Dhaka MRT Card.
 
-The FeliCa NFC Card stores the last 10 transactions performed by your card. That part of the 
-memory is readable through NFC commands. Thus, the system does not require any API connectivity
-to servers. However, only limited to 10 transactions. 
+### Key Features
+
+* Reads the last 10 transactions from the FeliCa NFC Card
```
[the remark this answers is not in the document]
Comment (Nafisarkar) [reply README.md:16]: Fixed 👾

File: README.md [hunk 5]
```diff
@@ -1,15 +1,27 @@
...
+* No API connectivity required, ensuring a seamless user experience
+* Limited to 10 transactions; for more, an official solution from DMTCL is necessary
+
+### Instructions
+* Ensure your phone supports NFC.
+* Place the MRT pass near the back of your phone properly for a successful scan.
+
+### Stay Updated
+Want to receive notifications when updates are available? Until approval on the Play Store, all updates will be announced on this WhatsApp channel! <br>
+[![Join WhatsApp Channel](https://shields.io/badge/Join-WhatsApp%20Channel-green?style=for-the-badge&logo=whatsapp&logoColor=white)](https://chat.whatsapp.com/KY8xK0qRm4t4l0q4qf4s)
```
[the remark this answers is not in the document]
Comment (Nafisarkar) [reply README.md:26]: Missed one . Destroyed ! 💥

File: README.md [hunk 6]
```diff
@@ -1,15 +1,27 @@
...
-to servers. However, only limited to 10 transactions. 
+### Key Features
+
+* Reads the last 10 transactions from the FeliCa NFC Card
+* No API connectivity required, ensuring a seamless user experience
+* Limited to 10 transactions; for more, an official solution from DMTCL is necessary
+
+### Instructions
+* Ensure your phone supports NFC.
+* Place the MRT pass near the back of your phone properly for a successful scan.
```
[the remark this answers is not in the document]
Comment (Nafisarkar) [reply README.md:22]: How about a gif ?

**Comment by aniruddha-adhikary** [comment]
Amazing work @Nafisarkar. Thank you so much for taking the time to organize all of this. Appreciate!

**Review by aniruddha-adhikary — COMMENTED** [review]
Approved

**Pull request merged**